testpilot/frontend/urls.py

```diff
@@ -1,8 +1,10 @@
-from django.conf.urls import url, patterns
+from django.conf.urls import patterns, url
 
 from . import views
 
+
 urlpatterns = patterns(
     '',
-    url(r'(?P<url>.*)', views.index, name="frontend_index")
+    url(r'^contribute\.json$', views.contribute_json, name='contribute_json'),
```

Comment:
Might be better to add this to [`base/urls.py`](https://github.com/mozilla/testpilot/blob/master/testpilot/base/urls.py), where we've accumulated some other general project views. The `frontend` app has so far been mainly for the ampersand-based client frontend